task_spec/spec/acceptance/init_spec.rb

```diff
@@ -147,9 +159,10 @@ def bolt_on(target, type, action, object,
       File.open(inventory_path, 'w') {|fh| fh.write(inventory.to_json)}
       File.open(config_path, 'w') {|fh| fh.write("---\n")}
       command += ['--inventoryfile', inventory_path]
-      command += ['--configfile', inventory_path]
+      command += ['--configfile', config_path]
```

Comment:
Was this set to `inventory_path` on purpose?

Reply:
Nope